legal-api/migrations/versions/004f7558063f_second.py

```diff
@@ -653,12 +686,17 @@ def upgrade():
     op.create_table('dc_issued_business_user_credentials',
     sa.Column('id', sa.Integer(), nullable=False),
     sa.Column('user_id', sa.Integer(), nullable=False),
-    sa.Column('legal_entity_id', sa.Integer(), nullable=False),
+    sa.Column('legal_entity_id', sa.Integer(), nullable=True),  # Allow null values
+    sa.Column('alternate_name_id', sa.Integer(), nullable=True),  # Allow null values
     sa.ForeignKeyConstraint(['legal_entity_id'], ['legal_entities.id']),
+    sa.ForeignKeyConstraint(['alternate_name_id'], ['alternate_names.id']),
     sa.ForeignKeyConstraint(['user_id'], ['users.id']),
     sa.PrimaryKeyConstraint('id')
     )
 
+    # Add a check constraint to ensure at least one of legal_entity_id or alternate_name_id is non-null
+    op.execute("ALTER TABLE dc_issued_business_user_credentials ADD CONSTRAINT at_least_one_non_null CHECK (legal_entity_id IS NOT NULL OR alternate_name_id IS NOT NULL)")
```

Comment:
`legal_entity_id` was previous non-nullable but with addition of `alternate_name_id`, made this field nullable.

Added a constraint to ensure at least `legal_entity_id` or `alternate_name` is non-null